c_api/src/c_api_test_utils.cpp

```diff
@@ -48,5 +48,62 @@ void check_runtime_error(TiRuntime runtime) {
 #endif
 }
 
+static void float32(float *__restrict out, const uint16_t in) {
```

Comment:
I've already implemented a similar function `half_to_float` in `taichi/util/bit.h`. You can use that and add `float_to_half` there?
https://github.com/taichi-dev/taichi/blob/4b385780fbb9f51fc2d80f021cc6937afd879214/taichi/util/bit.h#L28-L48

Reply:
Ah... I just realized I am not able to use any internal taichi utils in C-API tests, because C-API tests only link with `libtaichi_c_api.so` which only export symbols defined in `c_api/include/...`.

Reply:
In general, C-API tests are designed to behave as if external customers are using Taichi's `libtaichi_c_api.so`, and therefore helper functions in `c_api_test_utils.cpp` are utilities to be implemented in the customer's code.